app/scripts/MapboxTilesTrack.js

```diff
@@ -9,17 +9,17 @@ class MapboxTilesTrack extends OSMTilesTrack {
    * @param server: The server to pull tiles from.
    * @param tilesetUid: The data set to get the tiles from the server
    */
-  constructor(scene, options, animate, accessToken) {
+  constructor(context, options) {
+    super(context, options);
+
     // Force Mapbox and OpenStreetMaps copyright
-    // options.name = `© Mapbox © OpenStreetMap${options.name ? `\n${options.name}` : ''}`;
-    super(scene, options, animate);
+    this.options.name = `© Mapbox © OpenStreetMap${options.name ? `\n${options.name}` : ''}`;
```

Comment:
Ok, so imagine that the track is initially loaded: this.options.name is equal to ''. 

It goes through this chunk of code, is exported and loaded again. Now this.options.name is equal to '(c) Mapbox (c) OSM, etc...'. But the code in line 16 add '(c) Mapbox (c) OSM' again. So now it's there twice.

After the next export and load, the string is appended again. Do you see what I mean?

Let's just get rid of the line.

Reply:
Oh sorry, I didn't pay attention to the `${options.name ? `\n${options.name}` : ''}` is that new? Getting rid of it could be dangerous because people can remove a copyright information the legally should be there (just saying). Does it make sense to put this concatenation into the renderer so it's not exported?